### PR TITLE
Add new frame format view to help understand

### PR DESCRIPTION
**Pull request by MichelAraujo**
### Description
Just new point of the view for help understand frame format

### Motivation
Help other understand frame format more easily

### CODE REVIEW

**Comment by wbamberg** [comment]
Thank you for your PR, and sorry to take a while to look at it.

I'm not actually sure about this change though. I think numbering bytes gets confusing, because of the variable size of the payload length and the optionality of the masking key.

I know you say "8-bits payload length example" but I think this is a little hard to understand, and you don't mention masking.

Also, the diagram for bytes 3-6 (masking) are a lot of space without much real content.

If you wanted to summarise the format and map it to bytes, perhaps you could say something like:

```
- First byte:
       0 FIN
       1 RSV1
       2 RSV2
       3 RSV3
       4-7 OPCODE
- Bytes 2-10: payload length (see [Decoding Payload Length](#decoding_payload_length))
- If masking is used, next 4 bytes contain masking key (see [Reading and unmasking the data](#reading_and_unmasking_the_data))
- All subsequent bytes are payload
```

**Comment by MichelAraujo** [comment]
> Thank you for your PR, and sorry to take a while to look at it.
> 
> I'm not actually sure about this change though. I think numbering bytes gets confusing, because of the variable size of the payload length and the optionality of the masking key.
> 
> I know you say "8-bits payload length example" but I think this is a little hard to understand, and you don't mention masking.
> 
> Also, the diagram for bytes 3-6 (masking) are a lot of space without much real content.
> 
> If you wanted to summarise the format and map it to bytes, perhaps you could say something like:
> 
> ```
> - First byte:
>        0 FIN
>        1 RSV1
>        2 RSV2
>        3 RSV3
>        4-7 OPCODE
> - Bytes 2-10: payload length (see [Decoding Payload Length](#decoding_payload_length))
> - If masking is used, next 4 bytes contain masking key (see [Reading and unmasking the data](#reading_and_unmasking_the_data))
> - All subsequent bytes are payload
> ```
Great! 
I loved you summary suggestion, you right that my exemple is little dirty with all byte without real content, I tried representation the payload how a strip of the byte but is hard to make it clean.
Well, I implementate you suggestion and maybe can helper other to understand more easy.

File: files/en-us/web/api/websockets_api/writing_websocket_servers/index.md [hunk 1]
```diff
@@ -105,6 +105,21 @@ Frame format:
...
+       0 FIN
+       1 RSV1
+       2 RSV2
+       3 RSV3
+       4-7 OPCODE
+- Bytes 2-10: payload length (see [Decoding Payload Length](#decoding_payload_length))
+- If masking is used, next 4 bytes contain masking key (see [Reading and unmasking the data](#reading_and_unmasking_the_data))
+- All subsequent bytes are payload
+```
+
```
Comment (wbamberg) [inline files/en-us/web/api/websockets_api/writing_websocket_servers/index.md:122]:
```suggestion
This means that a frame contains the following bytes:

- First byte:
  - bit 0: FIN
  - bit 1: RSV1
  - bit 2: RSV2
  - bit 3: RSV3
  - bits 4-7 OPCODE
- Bytes 2-10: payload length (see [Decoding Payload Length](#decoding_payload_length))
- If masking is used, the next 4 bytes contain the masking key (see [Reading and unmasking the data](#reading_and_unmasking_the_data))
- All subsequent bytes are payload
```

File: files/en-us/web/api/websockets_api/writing_websocket_servers/index.md [hunk 2]
```diff
@@ -105,6 +105,17 @@ Frame format:
...
+- First byte:
+  - bit 0: FIN
+  - bit 1: RSV1
+  - bit 2: RSV2
+  - bit 3: RSV3
+  - bits 4-7 OPCODE
+- Bytes 2-10: payload length (see [Decoding Payload Length](#decoding_payload_length))
+- If masking is used, the next 4 bytes contain the masking key (see [Reading and unmasking the data](#reading_and_unmasking_the_data))
+- All subsequent bytes are payload
```
Comment (wbamberg) [inline files/en-us/web/api/websockets_api/writing_websocket_servers/index.md:118]:
```suggestion
- All subsequent bytes are payload

```

**Review by wbamberg — APPROVED** [review]
👍 thank you @MichelAraujo !

**Pull request merged**